src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_COST = 40;
+  const MANY_DAYS = 7;
+  const BIG_DISCOUNT = 50;
+  const FEW_DAYS = 3;
+  const SMALL_DISCOUNT = 20;
+
+  let totalCost = DAILY_COST * days;
```

Comment:
```suggestion
  const totalCost = DAILY_COST * days;
```
Prefer const over let